NameLayerMain/src/vg/civcraft/mc/namelayer/gui/GUIGroupOverview.java

```diff
@@ -78,14 +77,14 @@ public void clicked(Player p) {
 					NameLayerPlugin.log(Level.INFO,
 							p.getName() + " turned autoaccept for invites off "
 									+ "via gui");
-					NameLayerPlugin.getGroupManagerDao().removeAutoAcceptGroup(p.getUniqueId());
+					NameLayerPlugin.getGroupManagerDao().removeAutoAcceptGroupAsync(p.getUniqueId());
```

Comment:
That is fascinating ... in that case, this remains, but perhaps add a TODO line to remind ourselves of this finding?